webpack.config.js

```diff
@@ -9,6 +9,7 @@ Encore
     .disableSingleRuntimeChunk()
     .enableSourceMaps(!Encore.isProduction())
     .enableVersioning(Encore.isProduction())
+    .enablePostCssLoader()
```

Comment:
Isn‘t there a `configurePostCssLoader()` method that makes adding a `postcss.config.js` file redundant?

Reply:
<img width="728" alt="" src="https://github.com/user-attachments/assets/8b15d809-b981-4dfa-b41f-85e36b4bb66d">

Reply:
Done in e8ec9fb5ef5f31938ff8a801c4d6ad504dd3da40.